lib/tasks/json_web_token.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class JsonWebToken
+  SECRET_KEY = Rails.application.secrets.secret_key_base.to_s
+  def self.encode(payload, exp = 24.hours.from_now)
+  payload[:exp] = exp.to_i
+  JWT.encode(payload, SECRET_KEY)
+  end
+
+  def self.decode(token)
+  decoded = JWT.decode(token, SECRET_KEY).first
+  HashWithIndifferentAccess.new decoded
+  end
```

Comment:
I remember I did a comment about using class << self annotation. Is this ok?
What do you think about using the other syntax?

Reply:
Yes, I implemented your recommendation. I don't know why this file changed again. Sure it was my error merging branches.